ProcessMaker/spark-modeler
# Load call activity process list from API

Fixes #305.

**TODO**:
 * The generated XML for a _Call Activity_ seems valid, but results in an error, `The bpm definition is not valid`, when trying to save in BPM. See discussion in issue: https://github.com/ProcessMaker/modeler/issues/305#issuecomment-479166889.

**How to test:**
* Add a _Call Activity_ to the grid.
* Select the _Call Activity_ and the _Processes_ dropdown in the inspector panel should be populated with all processes from the API that have a start event.

<img width="537" alt="Screen Shot 2019-04-02 at 3 12 44 PM" src="https://user-images.githubusercontent.com/7561061/55430803-04103000-555d-11e9-8968-9a574ba6bb78.png">
<img width="545" alt="Screen Shot 2019-04-02 at 3 21 15 PM" src="https://user-images.githubusercontent.com/7561061/55430804-04103000-555d-11e9-9907-b64c6b1d3c52.png">
